site/cds_rdm/templates/semantic-ui/invenio_notifications/email/community-submission.submit.jinja

```diff
@@ -0,0 +1,58 @@
+{% set submission_request = notification.context.request %}
+{% set community = submission_request.receiver %}
+{% set creator = submission_request.created_by %}
+{% set record = submission_request.topic %}
+{% set request_id = submission_request.id %}
+{% set recipient_full_name = recipient.data.profile.full_name %}
+{% set help_url = "https://cern.service-now.com/service-portal?id=service_element&name=CDS-Service" %}
+{% set community_title = community.metadata.title %}
+{% set record_title = record.metadata.title %}
+{% set creator_name = creator.username %}
+
+{# TODO: use request.links.self_html when issue issue is resolved: https://github.com/inveniosoftware/invenio-rdm-records/issues/1327 #}
+{% set request_link = "{ui}/me/requests/{id}".format(
+    ui=config.SITE_UI_URL, id=request_id
+    )
+%}
+
+{%- block subject -%}
+    [CDS] New Submission: "{{ record_title }}" in {{ community_title }} community
+{%- endblock subject -%}
+
+{%- block html_body -%}
+<p>Dear {{ recipient_full_name }},</p>
```

Comment:
I am wondering if also community's curators/owners will receive this: in this case, I would remove this line, because the receiver might not be the recipient.
Do we simply put:
```
Hello,
```
?

Reply:
Each curator will receive the email with his own name on it